_layouts/home.html

```diff
@@ -0,0 +1,42 @@
+---
+layout: articles
+titles:
+  # @start locale config
+  en      : &EN       Home
+  en-GB   : *EN
+  en-US   : *EN
+  en-CA   : *EN
+  en-AU   : *EN
+  zh-Hans : &ZH_HANS  主页
+  zh      : *ZH_HANS
+  zh-CN   : *ZH_HANS
+  zh-SG   : *ZH_HANS
+  zh-Hant : &ZH_HANT  主頁
+  zh-TW   : *ZH_HANT
+  zh-HK   : *ZH_HANT
+  ko      : &KO       홈
+  ko-KR   : *KO
+  fr      : &FR       Accueil
+  fr-BE   : *FR
+  fr-CA   : *FR
+  fr-CH   : *FR
+  fr-FR   : *FR
+  fr-LU   : *FR
+  # @end locale config
+show_title: false
+articles:
+  data_source: paginator.posts
+  article_type: BlogPosting
+  show_cover: true
```

Comment:
I want to enable the cover image by using setting `show_cover: true`. But it's not enough for showing cover images:

![Screenshot 2021-02-12 at 10 59 32](https://user-images.githubusercontent.com/10179217/107754328-9c62c880-6d21-11eb-8a8b-7dddc936ec4e.png)

Reply:
It's working now. I need to add `cover` variable in the Page Layout as indicated by the documentation above. Posts without the `cover` variable won't have the cover:

![Screenshot 2021-02-12 at 11 26 11](https://user-images.githubusercontent.com/10179217/107756983-4001a800-6d25-11eb-8fb0-72b9630be381.png)